src/blocks/buy-link.css

```diff
@@ -0,0 +1,22 @@
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: bold;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.card__buy:hover {
+  height: 38px;
```

Comment:
no, you should not change it here, use `box-sizing: border-box` so you wouldn't need to calculate it on hover
```suggestion
```